test/mocha.opts

```diff
@@ -1,4 +1,4 @@
---timeout 1200s
+--timeout 1800s
```

Comment:
Can we somehow limit this timeout only for respective tests where we need more time for blocks?

Reply:
I am not sure we can dynamically change the timeout, however, we can use the timeout at `it` level, but not sure if wanna use it at `it` level.

Reply:
@nazarhussain I tried introducing the `this.timeout(1800000)` at `suite level` on my local and then it doesn't seem to work, so if you have any input gladly welcomed 👍

Reply:
👍